docs/reference/configuration.md

```diff
@@ -1,75 +1,99 @@
 ---
 layout: default
-title: Configuration
-description: Configuring lakeFS is done using a YAML configuration file. This reference uses `.` to denote the nesting of values.
+title: lakeFS Server Configuration
```

Comment:
"lakeFS" here is redundant. We can go with "Server Configuration", "Configuring the Server" or leave the previous title.

Reply:
ack, thanks.